cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -30,7 +33,7 @@ class Global {
 }
 
 // curl command example for running this plugin (Prior Artifactory 5.x, use pipe '|' and not semi-colons ';' for parameters separation).
-// curl -i -uadmin:password -X POST "http://localhost:8081/artifactory/api/plugins/execute/cleanup?params=months=1;repos=libs-release-local;dryRun=true;paceTimeMS=2000;disablePropertiesSupport=true"
+// curl -i -uadmin:password -X POST "http://localhost:8081/artifactory/api/plugins/execute/cleanup?params=months=1;repos=libs-release-local;dryRun=true;paceTimeMS=2000;disablePropertiesSupport=true,keepRelease=true"
```

Comment:
Params are separated by semicolons, it won't work with a comma.

Reply:
ouch! my mistake! i will fix it tomorrow